app/controllers/concerns/admin_session_timeout.rb

```diff
@@ -15,7 +15,7 @@ def admin_signed_in?
   end
 
   def admin_session_timed_out?
-    admin_signed_in? && session[:last_seen_at] < admin_timeout_in_minutes.minutes.ago
+    admin_signed_in? && session[:admin_last_seen_at] < admin_timeout_in_minutes.minutes.ago
```

Comment:
When we ship this, any already signed-in admin users will get a 500 error because `session[:admin_last_seen_at]` will be `nil` and `nil < something` will blow up. We should be aware of this and do what we can to minimise disruption and confusion for claim checkers.

Reply:
How about safe navigation here? 

```ruby
admin_signed_in? && session[:admin_last_seen_at].&< admin_timeout_in_minutes.minutes.ago
```

Reply:
That looks v.icky, especially as it should only be a temporary issue until their sessions are updated. I think it might be enough to either do the deploy when server operators aren't around, or ask them to sign-out whilst we do the deploy.

Reply:
Yeah, hence my asking 😃. It's probably not worth the long-term ickyness for the small benefit we get